docker-compose.yml

```diff
@@ -230,7 +230,29 @@ services:
         - "dockerhost:${DOCKER_HOST_IP}"
       networks:
         - backend
-### Laravel Horizon ############################################
+
+
+### NODEJS ##############################################
+    nodejs:
+      build:
+        context: ./nodejs
+        args:
+          - LARADOCK_NODE_VERSION=${NODE_VERSION}
+      volumes:
+        - ${APP_CODE_PATH_HOST}:${APP_CODE_PATH_CONTAINER}
+      expose:
+        - "3000-3010:3000-3010"
```

Comment:
Hi @zuohuadong 

travis [build](https://travis-ci.org/laradock/laradock/jobs/574691801#L1387) is failed, because expose format is not valid

please change `3000-3010:3000-3010` to `3000-3010`